core/setup.py

```diff
@@ -69,8 +69,8 @@ def read(fname):
         'isodate>=0.6,<0.7',
         'json-rpc>=1.12,<2',
         'werkzeug>=0.15,<2.0',
-        'dataclasses==0.6;python_version<"3.7"',
-        'hologram==0.0.13',
+        'dataclasses==0.7;python_version<"3.7"',
```

Comment:
also actually I'm not sure if we should pin strictly to 0.7 here or let it be like the hologram depenency which is `>=0.6,<0.9`

Reply:
I agree with having this match the hologram dependency exactly: `'dataclasses>=0.6,<0.9;python_version<"3.7"'` feels right.